app/controllers/pending-user-updates.js

```diff
@@ -49,7 +49,7 @@ export default Controller.extend({
     });
   }),
 
-  displayedUpdates: computed('allUpdates.@each.user.fullName', 'filter', 'offset', 'limit', function(){
+  displayedUpdates: computed('allUpdates.@each.user.fullName', 'filter', 'offset', 'limit', 'deletedUpdates.[]',  function(){
```

Comment:
corrected. that didn't do anything. listening for changes to `deletedUpdates` array is still they key to fixing this issue.